### PR TITLE
Add column description for some system tables

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs <crazycs520@gmail.com>

### What is changed, added or deleted? (Required)

As title said.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@crazycs520 Please involve a technical review.

File: system-tables/system-table-information-schema.md [hunk 1]
```diff
@@ -281,6 +281,40 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
 +----+------+------+--------------------+---------+------+-------+---------------------------+-----+
 ```
 
+`PROCESSLIST` 表各列的含义如下：
+
+* ID：客户连接 ID。
+* USER：执行当前 PROCESS 的用户名。
+* HOST： 客户连接的地址。
```
Comment (djshow832) [inline system-tables/system-table-information-schema.md:288]:
```suggestion
* HOST：客户连接的地址。
```

File: system-tables/system-table-information-schema.md [hunk 2]
```diff
@@ -356,27 +390,14 @@ SELECT * FROM schemata;
 5 rows in set (0.00 sec)
 ```
 
-## CLUSTER_PROCESSLIST
-
-`CLUSTER_PROCESSLIST` 是 `PROCESSLIST` 对应的集群系统表，用于查询集群中所有 TiDB 节点的 `PROCESSLIST` 信息。表结构上比 `PROCESSLIST` 多一列 `INSTANCE`，表示该行数据来自的 TiDB 节点地址。
+`SCHEMATA` 表各列含义如下：
```
Comment (crazycs520) [inline system-tables/system-table-information-schema.md:393]:
```suggestion
`SCHEMATA` 表各列含义如下：

```

**Comment by crazycs520** [comment]
@TomShawn PTAL

File: system-tables/system-table-information-schema.md [hunk 3]
```diff
@@ -333,6 +333,40 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
...
+* DB：当前连接的默认数据库名。
+* COMMAND：当前 PROCESS 执行的命令类型。
+* TIME：当前 PROCESS 的已经执行的时间，单位是秒。
+* STATE：当前连接的状态。
+* INFO：正在处理的请求语句。
+* MEM：正在处理的请求已使用的内存，单位是 byte。
+
+## CLUSTER_PROCESSLIST
+
+`CLUSTER_PROCESSLIST` 是 `PROCESSLIST` 对应的集群系统表，用于查询集群中所有 TiDB 节点的 `PROCESSLIST` 信息。表结构上比 `PROCESSLIST` 多一列 `INSTANCE`，表示该行数据来自的 TiDB 节点地址。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:350]:
```suggestion
`CLUSTER_PROCESSLIST` 是 `PROCESSLIST` 对应的集群系统表，用于查询集群中所有 TiDB 节点的 `PROCESSLIST` 信息。`CLUSTER_PROCESSLIST` 表结构上比 `PROCESSLIST` 多一列 `INSTANCE`，表示该行数据来自的 TiDB 节点地址。
```

File: system-tables/system-table-information-schema.md [hunk 4]
```diff
@@ -333,6 +333,40 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
...
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT * FROM information_schema.cluster_processlist;
+```
+
+```
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:358]:
```suggestion
```sql
```

File: system-tables/system-table-information-schema.md [hunk 5]
```diff
@@ -356,27 +390,13 @@ SELECT * FROM schemata;
 5 rows in set (0.00 sec)
 ```
 
-## CLUSTER_PROCESSLIST
-
-`CLUSTER_PROCESSLIST` 是 `PROCESSLIST` 对应的集群系统表，用于查询集群中所有 TiDB 节点的 `PROCESSLIST` 信息。表结构上比 `PROCESSLIST` 多一列 `INSTANCE`，表示该行数据来自的 TiDB 节点地址。
+`SCHEMATA` 表各列含义如下：
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:393]:
```suggestion
`SCHEMATA` 表各列字段含义如下：
```

**Pull request merged**